src/components/computed-properties.vue

```diff
@@ -78,25 +73,28 @@
         </template>
       </b-table>
       <template slot="modal-footer">
-        <span></span>
+        <span/>
       </template>
     </template>
 
     <template v-else>
       <form-input
         v-model="add.property"
         :label="$t('Property Name')"
-        name="property name"
-        validation="required"
+        :name="$t('Property Name')"
+        :validation="rulesValidation"
       />
+      <div v-if="existsProperty" class="invalid-feedback d-block">
```

Comment:
I think it would be cleaner to add this as a custom validator rule:
https://github.com/skaterdav85/validatorjs#register-custom-validation-rules